tests/copy.rs

```diff
@@ -168,32 +167,31 @@ fn urls(test: &str) -> Vec<Url> {
             .unwrap(),
 
         // Azure URLs
-        // These URLs use a SAS token that expires in 2050 and the default credentials of Azurite
-        format!("az://devstoreaccount1/{TEST_BUCKET_NAME}/1/{test}?sv=2018-03-28&spr=https%2Chttp&st=2025-08-11T15%3A49%3A59Z&se=2050-08-11T15%3A49%3A00Z&sr=c&sp=rcwl&sig=0UK1EckCkj0k8Xi7s7yjB5QpjZa%2FVUmtd906SWAtO%2FM%3D").parse().unwrap(),
-        format!("http://devstoreaccount1.blob.core.windows.net.localhost:10000/{TEST_BUCKET_NAME}/2/{test}?sv=2018-03-28&spr=https%2Chttp&st=2025-08-11T15%3A49%3A59Z&se=2050-08-11T15%3A49%3A00Z&sr=c&sp=rcwl&sig=0UK1EckCkj0k8Xi7s7yjB5QpjZa%2FVUmtd906SWAtO%2FM%3D").parse().unwrap(),
+        format!("az://devstoreaccount1/{TEST_BUCKET_NAME}/1/{test}").parse().unwrap(),
+        format!("http://devstoreaccount1.blob.core.windows.net.localhost:10000/{TEST_BUCKET_NAME}/2/{test}").parse().unwrap(),
     ]
 }
 
-fn config() -> Config {
-    Config {
-        overwrite: true,
-        azure: AzureConfig { use_azurite: true },
-        s3: S3Config {
-            use_localstack: true,
-            auth: Some(S3AuthConfig {
-                access_key_id: "test".to_string(),
-                secret_access_key: "test".into(),
-            }),
-            ..Default::default()
-        },
-        ..Default::default()
-    }
+fn config(overwrite: bool) -> Config {
+    Config::builder()
+        .with_overwrite(overwrite)
+        .with_azure(AzureConfig::default().with_use_azurite(true).with_auth(
+            "devstoreaccount1",
+            "Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/\
```

Comment:
Just to be sure, this is the public key that is included with Azurite, correct?

Reply:
That's correct.  I'll add a comment to clarify so that folks seeing an access key here don't think it's a real one.